src/ui/VoiceMessageInput/index.scss

```diff
@@ -0,0 +1,176 @@
+@import '../../styles/variables';
+
+.sendbird-voice-message-input {
+  position: relative;
+  display: inline-flex;
+  flex-direction: column;
+  width: 100%;
+  height: 110px;
+  padding-top: 22px;
+  padding-left: 24px;
+  padding-right: 24px;
+  box-sizing: border-box;
+}
+
+.sendbird-voice-message-input__indicator {
+  position: relative;
+  display: inline-flex;
+  width: 100%;
+}
+.sendbird-voice-message-input__indicator__progress-bar {
+  position: relative;
+  display: inline-flex;
+  width: 100%;
+  border-radius: 28px;
+  overflow: hidden;
+}
+.sendbird-voice-message-input__indicator__on-rec {
+  position: absolute;
+  top: 14px;
+  right: 53px;
+}
+.sendbird-voice-message-input__indicator__playback-time {
+  position: absolute;
+  top: 10px;
+  right: 16px;
+}
+
+.sendbird-voice-message-input__controler {
+  position: relative;
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 14px;
+  height: 34px;
+  min-height: 34px;
+}
+.sendbird-voice-message-input__controler__main {
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  // icon
+  width: 34px;
+  height: 34px;
+  min-width: 34px;
+  min-height: 34px;
+  border-radius: 50%;
+  @include themed() {
+    background-color: t(bg-1);
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.sendbird-voice-message-input__controler__submit {
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 34px;
+  height: 34px;
+  min-width: 34px;
+  min-height: 34px;
+  border-radius: 50%;
+  @include themed() {
+    background-color: t(primary-3);
+  }
+  &:hover {
+    cursor: pointer;
+  }
+}
+.sendbird-voice-message-input__controler__submit.voice-message--disabled {
+  @include themed() {
+    background-color: t(bg-1);
+  }
+  &:hover {
+    cursor: default;
+  }
+}
+
+// Flicker
+@keyframes flickerAnimation {
+  0%   { opacity:0; }
+  50%  { opacity:1; }
+  100% { opacity:0; }
+}
+@-o-keyframes flickerAnimation {
+  0%   { opacity:0; }
+  50%  { opacity:1; }
+  100% { opacity:0; }
+}
+@-moz-keyframes flickerAnimation {
+  0%   { opacity:0; }
+  50%  { opacity:1; }
+  100% { opacity:0; }
+}
+@-webkit-keyframes flickerAnimation {
+  0%   { opacity:0; }
+  50%  { opacity:1; }
+  100% { opacity:0; }
+}
```

Comment:
I think you dont need to do this, there is SCSS auto-prefixer, we can check after release.. Lets make a ticket

Reply:
hmm.. we already do autoprefix
<img width="559" alt="Screen Shot 2023-02-21 at 5 49 43 PM" src="https://user-images.githubusercontent.com/11382805/220294930-f4557834-ab11-42b2-82f6-b97763cc627c.png">

<img width="589" alt="Screen Shot 2023-02-21 at 5 50 00 PM" src="https://user-images.githubusercontent.com/11382805/220294959-ebc930b3-e7cc-4a9e-8cb8-7b632a457d6a.png">